src/components/pages/tutorials/Tutorials.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Icon from '../../layout/Icon/Icon'
+
+const Tutorials = () => (
+  <main className="tutorials">
+    <Icon
+      alt="calendrier"
+      className="calendar"
+      svg="calendar"
+    />
+    <p className="first-tutorial-text">
+      <span>
+        {"À partir d'aujourd'hui, tu as "}
+      </span>
+      <span className="text-highlight text-2-ans">
+        {'2'}
+        &nbsp;
+        {'ans'}
```

Comment:
```suggestion
        {`2${NON_BREAKING_SPACE}ans`}
```
J'aime voir des nbspc là où il va bien d'en mettre <3